test cases/failing/98 number in combo/test.json

```diff
@@ -1,5 +1,5 @@
 {
   "stdout": [
-    { "line": "test cases/failing/98 number in combo/meson.build:1:0: ERROR: Value \"1\" (of type \"number\") for combo option \"Optimization level\" is not one of the choices. Possible choices are (as string): \"plain\", \"0\", \"g\", \"1\", \"2\", \"3\", \"s\"." }
+    { "line": "test cases/failing/98 number in combo/meson.build:1:0: ERROR: Value \"1\" (of type \"number\") for option('optimization') is not one of the choices. Possible choices are (as string): \"plain\", \"0\", \"g\", \"1\", \"2\", \"3\", \"s\"." }
```

Comment:
This is a pretty weird way to spell the option name in an error message. I would much prefer something like `...for option "optimization"...` instead of `...for option('optimization')...`. This makes constructing the error message a bit more verbose, but I think user experience should be valued more than conciseness in the code.

Reply:
I agree. Does it look better now?

Reply:
I'll really like to have quotes around the options names, otherwise the error messages may get hard to grammatically parse.

Reply:
ok. Should be fixed now.